app/services/resources/search.rb

```diff
@@ -1,18 +1,34 @@
 module Resources
   class Search
-    SEARCH_COLUMNS = %i(
-      name
-      short_description
-      long_description
-      website
-    ).freeze
+    SEARCH_CONFIG = 'english'.freeze
```

Comment:
With this passed to `to_tsvector` and `plainto_tsquery`, Postgres will actually normalize the fields nad query with an English stemmer. My local database uses English by default, but it seems Postgres in RDS has no default configuration, so we have to specify it explicitly.